exercises/.dir-locals.el

```diff
@@ -0,0 +1,5 @@
+;;; Directory Local Variables
```

Comment:
I *think* so, but I'm on the fence about it, too. Generally editor settings (cf https://editorconfig.org) are shared at the project level to keep style consistent. I saw a lot of variation with tabs/spaces while I was editing, so hopefully this phases that out without being too intrusive.

Reply:
One would think it would be safe to assume that `.dir-locals.el` should work in that regard, as one would think that anyone making contributions to the elisp track would be using Emacs as their primary editor, however I wonder if it possible to get a reasonable `editorconfig` working for this project. 

That said lets not overcomplicate things in this PR, and just go ahead and merge as is.

Reply:
I also think it's safe to assume emacs will be the editor :-) editorconfig requires its own setup within emacs, so that just adds an unnecessary step.